### DISCUSSION TITLE
Won't load properly

### DISCUSSION THREAD

**Discussion by jewellui** [valinet/ExplorerPatcher#3974] [Q&A]
Hi

I am new to ExplorerPatcher. 

When I open the .exe my taskbar disappears and my background changes to white, I can't see any of the desktop icons shortcuts. Any advice?

I wonder if one of my other taskbar programs is interfering. I turned those off but I still had the same problem.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.  Please include if you are running x86-64 or Arm.  And what language your windows is in.

And are you trying to install this version of EP? 22621.4317.67.1: https://github.com/valinet/ExplorerPatcher/releases

**Reply by jewellui** [reply]
Hi

Windows 11 Home
Version	10.0.22631 Build 22631
x64
English UK

Yes 22621.4317.67.1

**Reply by pyrates999** [reply]
You didn't post your full windows build number.  It should be in the format of #.#

**Reply by jewellui** [reply]
22631.4460

**Comment by pyrates999** [comment]
make sure you've ran the powershell commands here: https://github.com/valinet/ExplorerPatcher/releases

if you run a third party antivirus, make sure to exclude those files and directories.

then attempt to install EP.

**Reply by jewellui** [reply]
Thanks for getting back to me.

I tried pasting the first line with no luck, I'm unfamiliar with Powershell so I don't know if I am doing it wrong. I attached a screenshot.

![2024-11-27 22_19_02-Select Administrator_ Windows PowerShell](https://github.com/user-attachments/assets/d7a69d78-d7ad-413d-92a0-57dc1ae1d867)

**Reply by pyrates999** [reply]
See the release notes here for the powershell script you can run that excludes all the directories necessary in windows defender: https://github.com/valinet/ExplorerPatcher/releases

The powershell script you must run with elevated privileges (right click on powershell and select run as administrator).

Future updates to EP won't be flagged then.

You can also set windows defender to exclude the directory that you manually download EP to so you can install it without windows defender blocking it.

if you get the error: 0x800106ba, it means Windows Defender is not active. You may have another antivirus program active, which means you need to add those files and directories to be excluded in your antivirus program.

**Reply by jewellui** [reply]
I was on Admin already. I did some Googling it seems the error is due to Defender not running which makes sense because I have Avast but I added the exceptions already just before.

I tried it again and surprisingly it worked this time. It seems it Avast preventing it from working the first time, thank you.

**Comment by jewellui** [comment]
I was having an issue with File Explorer randomly opening whenever I was watching a video on full screen, not sure if its a bug related to ExplorerPatcher or just a Windows 11 bug.

So I tried reinstalling ExplorerPatcher but now it no longer works again, the primary taskbar doesn't show up when it's running. It was good while it lasted...

**Reply by jewellui** [reply]
Ok I managed to get this working again.

I found the problem, the "Taskbar style" was set to "Windows 10" which for some reason causes my primary taskbar to not appear anywhere and breaks the other taskbars Start buttons. When I switched to the "Windows 11" the primary taskbar appears and works again. 

It would be helpful if ExplorerPatcher offered some way to go to properties menu without having to right-click on the taskbar, since in my situation there is no way to access and change the properties as far as I'm aware. A way to open the properties from the start menu or right-clicking on the desktop background would be helpful/

I think it was by chance I got to the properties area when I uninstalled ExplorerPatcher and also removed my old settings then when I reinstalled it, and restarted my computer a few times the notification to download Symbols appeared allowing me to access it.

Then after updating my Windows version to 23H2 I updated to ExplorerPatcher's latest version I noticed in the Tasbar Style options "Windows 10 (ExplorerPatcher)" was now a new choice, this fixed my primary taskbar.

**Reply by pyrates999** [reply]
to run the explorer patcher properties application, run this from the run menu:
`C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI`

This is also available in the start menu if you go to ExplorerPatcher -> Properties (ExplorerPatcher).

**Reply by jewellui** [reply]
Oh yes, I see you can access it on the start menu.

I suppose the problem in my situation, since the taskbar is not working I can not access the start menu to find it so I think there needs to be an easy way to access it.

**Reply by pyrates999** [reply]
Add the shortcut to your desktop then.

**Reply by jewellui** [reply]
What I mean is that there could well be others who encounter the same issue, especially their first time using it like I did, but give up. They wouldn't be adding the shortcut in advance or may not realise, so I think it makes sense for another shortcut. Anyway, hopefully, someone will find this post helpful.

**Reply by pyrates999** [reply]
So suggestion to add the shortcut for EP Properties application to the all users desktop?

**Reply by jewellui** [reply]
I guess that's the best option. I don't know anything about software, it would be even better if ExplorerPatcher could detect the taskbar was not showing.
